### DISCUSSION TITLE
I have a problem connecting to the ollama service

### DISCUSSION THREAD

**Discussion by lei1402** [LibreChat-AI/docs#74] [Q&A]
https://www.librechat.ai/docs/configuration/librechat_yaml/ai_endpoints/ollama  I deployed librechat and I want to connect to my own ollama, my ollama address is 10.18.32.87:11434. I started using the default docker-compose.yml. I modified librechat.yaml but I can't see ollama in librechat. Should I modify it in .env? Is there any configuration you can refer to? thank you for your support.

**Comment by fuegovic** [comment]
Did you follw this step to make the yaml file visible to the docker container? (using the override file)
https://www.librechat.ai/docs/configuration/librechat_yaml/setup#docker-setup
